BridgeAppSDK/SBANavigableOrderedTask.swift

```diff
@@ -116,11 +116,11 @@ public class SBANavigableOrderedTask: ORKOrderedTask {
         // Look for step in the ordered steps and lop off everything after this one
         if let previousIdentifier = step?.identifier,
             let idx = self.orderedStepIdentifiers.indexOf(previousIdentifier) where idx < self.orderedStepIdentifiers.endIndex {
-                self.orderedStepIdentifiers.removeRange(Range(start: idx.advancedBy(1), end: self.orderedStepIdentifiers.endIndex))
+                self.orderedStepIdentifiers.removeRange(idx.advancedBy(1) ..< self.orderedStepIdentifiers.endIndex)
```

Comment:
wouldn't this mean remove from idx + 1 to endIndex - 1 (..<)? Or does that operator mean "up to and including"?